composer.json

```diff
@@ -21,7 +21,8 @@
         "ptrofimov/xpmock": "^1.1",
         "dhii/php-cs-fixer-config": "^0.1",
         "codeclimate/php-test-reporter": "<=0.3.2",
-        "rebelcode/rcmod-wp-bookings-ui": "0.1-alpha13"
+        "rebelcode/rcmod-wp-bookings-ui": "0.1-alpha13",
+        "rebelcode/rcmod-eddbk-rest-api": "0.1-alpha8"
```

Comment:
Why not use the caret `^`? Is that even a good idea @XedinUnknown?

Reply:
I'm not sure how good of an idea this is for modules, especially those that are in alpha. But I think it's a good idea to prevent locks, where we have to update an irrelevant module, because a common dependency is updated. So, yea, carrot operator. 🥕